apps/src/gamelab/GameLabP5.js

```diff
@@ -62,30 +62,30 @@ GameLabP5.prototype.init = function (options) {
     /*
      * Copied code from p5 from redraw()
      */
-    var userSetup = this.setup || window.setup;
-    var userDraw = this.draw || window.draw;
+    const userSetup = this.setup || window.setup;
+    const userDraw = this.draw || window.draw;
     if (typeof userDraw === 'function') {
       this.resetMatrix();
       if (typeof userSetup === 'undefined') {
         this.scale(this.pixelDensity, this.pixelDensity);
       }
-      var self = this;
-      this._registeredMethods.pre.forEach(function (f) {
-        f.call(self);
-      });
+      const preMethods = this._registeredMethods.pre;
+      for (let i = 0; i < preMethods.length; i++) {
+        preMethods[i].call(this);
+      }
```

Comment:
This change and the one directly below I'm not _sure_ will have a big impact, but they seem worthwhile since they both exist inside our draw loop so they happen a lot.

I'm replacing a `forEach()` call with a regular `for` loop because it allows us to create one less function, and make `n` less function invocations, per draw call.  Because this eliminates a layer of scope we can also get rid of the unneeded `self` temporary variable.